flink-runtime/src/test/java/org/apache/flink/runtime/state/TestTaskStateManager.java

```diff
@@ -94,7 +95,6 @@ public void reportTaskStateSnapshots(
 		@Nullable TaskStateSnapshot acknowledgedState,
 		@Nullable TaskStateSnapshot localState) {
 
-
```

Comment:
unrelated change

Reply:
Do you mean it should go to a separate commit or PR?
I'd rather choose a cleaner git/PR history.

Reply:
To be strict, any unrelated formatting change should be a separate `hotfix` commit, not need a separate PR, can commit separately in this PR.

Because I was also told the same issue multiple times by other committers when submitting PR in the early period. So I just mention it, it is not mandatory from my side. :)

Reply:
I understand. Extracted into a separate commit.